src/app/player-creation/services/player-creation.service.ts

```diff
@@ -27,8 +30,9 @@ export class PlayerCreationService {
     const player = this.initializeBasicPlayer(name, roomId);
     this.playerService.add(player)
       .subscribe(playerId => {
```

Comment:
Ok, it has been some time since i played around with rxjs, but something like that wouldn't work? (I don't like the "subscribe" cascade)

```
this.playerService.add(player)
  .pipe(merge(playerId => {
     const playerInRoom = { playerId: playerId, roomId: roomId, isActive: true } as PlayerInRoom;
     this.playerInRoomService.add(playerInRoom)
   }))
 .subscribe(...)
```